sphinx_gallery/docs_resolv.py

```diff
@@ -49,7 +50,7 @@ def _get_data(url):
         else:
             raise RuntimeError('unknown encoding')
     else:
-        with open(url, 'r') as fid:
+        with codecs.open(url, mode='r', encoding='utf-8') as fid:
             data = fid.read()
```

Comment:
Can you try `open(url, 'rb') as fid` and `fid.read.decode('utf-8')` instead? Maybe it will fix Travis